src/modules/Crop/Module.js

```diff
@@ -27,6 +27,8 @@ module.exports = function CropModule(options, UI) {
 
     var step = this;
 
+    options.step.input = input.src;
```

Comment:
What does this line do?

Reply:
What this does is that it provides the original image to Ui.js so it can be used to estimate the area of crop in the Ui
https://github.com/publiclab/image-sequencer/blob/12c422a07b55233ae94e13fc3700da2101916c65/src/modules/Crop/Ui.js#L90

Reply:
Was this the line that was breaking the crop module?

Reply:
Yes!

Reply:
Ohk.